### PR TITLE
feat(alerts): Alert effects (MTC)

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [na] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing

### Description

This adds two supported service alert effects to the alerts editor (see screenshot):
- No effect
- Accessibility issue

image

### CODE REVIEW

File: lib/alerts/util/index.js [hunk 1]
```diff
@@ -54,7 +54,10 @@ export const EFFECTS = [
   'ADDITIONAL_SERVICE',
   'MODIFIED_SERVICE',
   'STOP_MOVED',
-  'OTHER_EFFECT'
+  'OTHER_EFFECT',
+  // Items introduced by MTC.
```
Comment (landonreed) [inline lib/alerts/util/index.js:58]: This comment is incorrect. These are items added in v2 of GTFS-rt: https://gtfs.org/reference/realtime/v2/#enum-effect

Comment (binh-dam-ibigroup) [reply lib/alerts/util/index.js]: Just pushed.

**Comment by evansiroky** [comment]
Although I like that this has a target of being merged to dev and I think we should be merging to dev for all MTC branches, I'll wait and see if Landon wanted to merge it into some other random MTC branch.

**Comment by landonreed** [comment]
Blocked from merge. Awaiting merge/deployment on June 17, 2020.

**Pull request merged**